docs/querying/lookups.md

```diff
@@ -62,6 +62,12 @@ SELECT
 FROM sales
 GROUP BY 1
 ```
+Lookup function also accepts the 3rd argument called `$replaceMissingValueWith` as constant string, if you value is missing given lookups for queried key then lookup function return result value from `replaceMissingValueWith`
```

Comment:
nit. The lookup function also accepts the 3rd argument called `$replaceMissingValueWith` as a constant string. If your value is missing a lookup for the queried key, the lookup function returns the result value from `replaceMissingValueWith`

Reply:
fixed